mobile/library/cc/engine_builder.cc

```diff
@@ -475,11 +475,15 @@ std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> EngineBuilder::generate
     MessageUtil::loadFromYaml((*filter).typed_config_, *native_filter->mutable_typed_config(),
                               ProtobufMessage::getStrictValidationVisitor());
 #else
+#ifdef ENVOY_ENABLE_FULL_PROTOS
```

Comment:
should this be an `elif`?  Or probably have logic elsewhere to make sure both enable_yaml and enable_full_protos can't be set together.

Reply:
hmmm, arguably but I think I'm going to go with the fact we're the only users of either and not worry if thta's ok